sample/src/main/java/org/walletconnect/example/dapp/DappFragment.kt

```diff
@@ -0,0 +1,22 @@
+package org.walletconnect.example.dapp
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import org.walletconnect.example.R
+import org.walletconnect.example.databinding.DappFragmentBinding
+
+class DappFragment : Fragment(R.layout.dapp_fragment) {
+
+    private lateinit var binding: DappFragmentBinding
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding = DappFragmentBinding.bind(view)
+    }
```

Comment:
Add a function setupToolbar()
```suggestion
        binding = DappFragmentBinding.bind(view)
        setupToolbar()
    }
```